code/game/objects/items/surgery_tray.dm

```diff
@@ -15,31 +15,38 @@
 
 	/// If true we're currently portable
 	var/is_portable = TRUE
-	/// List of things that we spawn containing
-	var/list/initial_contents = list(
-		/obj/item/blood_filter,
-		/obj/item/bonesetter,
-		/obj/item/cautery,
-		/obj/item/circular_saw,
-		/obj/item/clothing/mask/surgical,
-		/obj/item/hemostat,
-		/obj/item/razor/surgery,
-		/obj/item/retractor,
-		/obj/item/scalpel,
-		/obj/item/stack/medical/bone_gel,
-		/obj/item/stack/sticky_tape/surgical,
-		/obj/item/surgical_drapes,
-		/obj/item/surgicaldrill,
-	)
 
 /obj/item/surgery_tray/deployed
 	is_portable = FALSE
 
+/obj/item/surgery_tray/full
+
+/obj/item/surgery_tray/full/deployed
+	is_portable = FALSE
+
+/obj/item/surgery_tray/proc/PopulateContents()
+
+/obj/item/surgery_tray/full/PopulateContents()
+	new /obj/item/blood_filter(src)
+	new /obj/item/bonesetter(src)
+	new /obj/item/cautery(src)
+	new /obj/item/circular_saw(src)
+	new /obj/item/clothing/mask/surgical(src)
+	new /obj/item/hemostat(src)
+	new /obj/item/razor/surgery(src)
+	new /obj/item/retractor(src)
+	new /obj/item/scalpel(src)
+	new /obj/item/stack/medical/bone_gel(src)
+	new /obj/item/stack/sticky_tape/surgical(src)
+	new /obj/item/surgical_drapes(src)
+	new /obj/item/surgicaldrill(src)
+	update_appearance(UPDATE_OVERLAYS)
+
 /obj/item/surgery_tray/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/drag_pickup)
 	create_storage(storage_type = /datum/storage/surgery_tray)
-	populate_contents()
+	PopulateContents()
```

Comment:
why did you take a proc name in the correct snake_case and make it PascalCase?
put it back

Reply:
I'm more curious as to why surgery trays aren't a child of the storage type (which uses PascalCase)

Reply:
The storage type uses pascal case because it's old and doesn't follow our style guidelines.
This isn't a subtype of it because it gains basically nothing from being one.

Reply:
Changed.